packages/amplication-client/src/App.tsx

```diff
@@ -112,8 +130,9 @@ function App() {
       {workspaceUpgradeConfirmation && (
         <PlanUpgradeConfirmation
           isOpen={workspaceUpgradeConfirmation}
-          onConfirm={() => setWorkspaceUpgradeConfirmation(false)}
-          onDismiss={() => setWorkspaceUpgradeConfirmation(false)}
+          onConfirm={closeConfirmation}
+          onDismiss={closeConfirmation}
```

Comment:
I don't think the onDismiss should use the same event as onComfirm

Reply:
that's by intention, the idea is to close purchase confirmation.